core/lib/src/main/java/dev/enola/core/connector/enola_connector.proto

```diff
@@ -31,9 +31,27 @@ message AugmentRequest {
 message AugmentResponse {
   dev.enola.core.Entity entity = 1;
 }
+
+// Request all the entities related via 'relationship' to 'kind' in 'scope'.
+// Example would be: list all the book copies (kind) from a specific library
+// (scope) that are currently borrowed.
+message ListRequest {
+  // The entityKind to be returned.
+  dev.enola.core.ID kind = 1;
+  // The related entity should be returned
+  dev.enola.core.ID scope = 2;
+  // The relevant relationship, by key, as in entityKind.related
+  string relationship = 3;
```

Comment:
@gabrielh I have a thought a bit more about this, and I now see what you want to do here, but I think it would be better as follows, so I'll go ahead and merge this and then change it to this format (and then implement the next steps from #192), hope you don't mind:

```c
  // Filters on related entities.
  // The key is as in EntityKind or Entity's .related,
  // and the value is the ("scope") related entity to filter on.
  // If there is more than one, these are AND, not OR.
  map<string, ID> relatedFilter = 2;
```